packages/mermaid/src/mermaid.ts

```diff
@@ -68,6 +58,18 @@ const init = async function (
   }
 };
 
+/**
+ * Equivalent to {@link init()}, except an error will be thrown on error.
+ *
+ * @param config - **Deprecated** Mermaid sequenceConfig.
```

Comment:
**Possible bug** This parameter seems to do nothing. It's set as `mermaid.sequenceConfig = config;`, but I can't see where `sequenceConfig` is actually used.

Reply:
https://github.com/mermaid-js/mermaid/issues/3406 Same bug?

Reply:
Looks like it. According to the code, it's only supposed to set the sequenceConfig, but it doesn't seem to set the right sequenceConfig.